modules/calendar/calendar.php

```diff
@@ -952,51 +971,54 @@ function generate_post_li_html( $post, $post_date, $num = 0 ){
 				</div>
 			</div>
 		</li>
-		<?php
+			<?php
 
-		$post_li_html = ob_get_contents();
-		ob_end_clean();
+			$post_li_html = ob_get_contents();
+			ob_end_clean();
 
-		$post_li_cache = array(
-			'num' => $num,
-			'post_li_html' => $post_li_html,
-			'hidden' => $this->hidden,
+			$post_li_cache = array(
+				'num' => $num,
+				'post_li_html' => $post_li_html,
+				'hidden' => $this->hidden,
 			);
-		wp_cache_set( $cache_key, $post_li_cache, self::$post_li_html_cache_key );
-
-		return $post_li_html;
-
-	} // generate_post_li_html()
-
-	/**
-	 * get_inner_information description
-	 * Functionality for generating the inner html elements on the calendar
-	 * has been separated out so various ajax functions can reload certain
-	 * parts of an inner html element.
-	 * @param  array $ef_calendar_item_information_fields
-	 * @param  WP_Post $post
-	 * @param  array $published_statuses
-	 *
-	 * @since 0.8
-	 */
-	function get_inner_information( $ef_calendar_item_information_fields, $post ) {
-		?>
+			wp_cache_set( $cache_key, $post_li_cache, self::$post_li_html_cache_key );
+
+			return $post_li_html;
+		} // generate_post_li_html()
+
+		/**
+		 * get_inner_information description
+		 * Functionality for generating the inner html elements on the calendar
+		 * has been separated out so various ajax functions can reload certain
+		 * parts of an inner html element.
+		 * @param  array $ef_calendar_item_information_fields
+		 * @param  WP_Post $post
+		 * @param  array $published_statuses
+		 *
+		 * @since 0.8
+		 */
+		public function get_inner_information( $ef_calendar_item_information_fields, $post ) {
+			?>
 			<table class="item-information">
-				<?php foreach( $this->get_post_information_fields( $post ) as $field => $values ): ?>
+				<?php foreach ( $this->get_post_information_fields( $post ) as $field => $values ) : ?>
 					<tr class="item-field item-information-<?php echo esc_attr( $field ); ?>">
 						<th class="label"><?php echo esc_html( $values['label'] ); ?>:</th>
-						<?php if ( $values['value'] && isset($values['type']) ): ?>
-							<?php if( isset( $values['editable'] ) && $this->current_user_can_modify_post( $post ) ) : ?>
-								<td class="value<?php if( $values['editable'] ) { ?> editable-value<?php } ?>"><?php echo esc_html( $values['value'] ); ?></td>
-								<?php if( $values['editable'] ): ?>
+						<?php if ( $values['value'] && isset( $values['type'] ) ) : ?>
+							<?php if ( isset( $values['editable'] ) && $this->current_user_can_modify_post( $post ) ) : ?>
+								<td class="value
```

Comment:
This seems to add some newlines in the class attribute which seems off. Could we pre-calculate `editable-value`, something like this?

```php
<?php $editable_class = $values['editable'] ? 'editable-value' : ''; ?>

// ...

<td class="value <?php echo esc_attr( $editable_class ) ?>"><?php echo esc_html( $values['value'] ); ?></td>
```

Reply:
Done